pkg/virtctl/root.go

```diff
@@ -96,6 +86,19 @@ func NewVirtctlCommand() *cobra.Command {
 	return rootCmd
 }
 
+// GetProgramName returns the commmand name to display in help texts.
+// If `virtctl` is installed via krew to be used as a kubectl plugin, it's run via a symlink, so the basename then
+// is `kubectl-virt`. In this case we want to accomodate the user by adjusting the help text (usage, examples and
+// the like) by displaying `kubectl virt <command>` instead of `virtctl <command>`.
+// see https://github.com/kubevirt/kubevirt/issues/2356 for more details
+// see also templates.go
+func GetProgramName(binary string) string {
+	if strings.HasSuffix(binary, "-virt") {
+		return binary[0:len(binary)-len("-virt")] + " virt"
```

Comment:
How about `fmt.Sprintf("%s-virt", strings.TrimSuffix(binary, "-virt"))?

Reply:
Thanks for the suggestion, replaced.